Get mmkeys.dll out of the toplevel site-library dir.

--- origsrc/sonata-1.5.3/setup.py	2008-09-13 23:45:42.000000000 -0500
+++ src/sonata-1.5.3/setup.py	2009-03-08 14:32:27.906250000 -0500
@@ -65,7 +65,7 @@ setup(name='Sonata',
         packages=["sonata"],
         package_dir={"sonata": "sonata/"},
         ext_modules=[Extension(
-        "mmkeys", ["mmkeys/mmkeyspy.c", "mmkeys/mmkeys.c", "mmkeys/mmkeysmodule.c"],
+        "sonata/mmkeys", ["mmkeys/mmkeyspy.c", "mmkeys/mmkeys.c", "mmkeys/mmkeysmodule.c"],
         extra_compile_args=capture("pkg-config --cflags gtk+-2.0 pygtk-2.0").split(),
         extra_link_args=capture("pkg-config --libs gtk+-2.0 pygtk-2.0").split()
          ),],
--- origsrc/sonata-1.5.3/sonata/main.py	2008-09-13 23:45:42.000000000 -0500
+++ src/sonata-1.5.3/sonata/main.py	2009-03-08 14:36:46.000000000 -0500
@@ -118,7 +118,7 @@ if not skip_gui:
 	if not HAVE_GNOME_MMKEYS:
 		try:
 			# if not gnome 2.18+, mmkeys for everyone else
-			import mmkeys
+			from sonata import mmkeys
 			HAVE_MMKEYS = True
 		except:
 			HAVE_MMKEYS = False
